x-pack/plugins/security_solution/public/actions/show_top_n/cell_action/show_top_n.tsx

```diff
@@ -46,7 +47,7 @@ export const createShowTopNCellActionFactory = createCellActionFactory(
     getDisplayNameTooltip: ({ field }) => SHOW_TOP(field.name),
     isCompatible: async ({ field }) =>
       fieldHasCellActions(field.name) &&
-      !UNSUPPORTED_FIELD_TYPES.includes(field.type) &&
+      !UNSUPPORTED_FIELD_TYPES.includes(head(field.esTypes) ?? '') &&
```

Comment:
This change could introduce bugs if we don't provide `esTypes` to the `CellActions` calls.

Reply:
Are we always going to check only the first element in `esTypes`?